app/src/main/java/chat/rocket/android/api/rest/ServerPolicyApi.java

```diff
@@ -0,0 +1,19 @@
+package chat.rocket.android.api.rest;
+
+import android.support.annotation.NonNull;
+import org.json.JSONObject;
+
+public interface ServerPolicyApi {
+
+  void getApiInfoSecurely(@NonNull Callback callback);
+
+  void getApiInfoInsecurely(@NonNull Callback callback);
+
+  interface Callback {
```

Comment:
When/if we get a second REST call then that would be the right moment to extract it to its own class. This will probably be our only call outside the file upload stuff.

And we're discussing if we should create a method to obtain the same set of data.

About the `Rx` / `Bolts` stuff: that may work. I'll build something here.